app/controllers/planning_applications/assessment/local_policy_areas_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class LocalPolicyAreasController < AuthenticationController
+      before_action :set_planning_application
+      before_action :set_local_policy, only: %i[create destroy edit new show update]
+      before_action :set_local_policy_area, only: %i[edit show update]
+
+      def show
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def new
+        @local_policy_area = @local_policy.local_policy_areas.new
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def edit
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def destroy
+        @local_policy.local_policy_areas.destroy(params[:id])
+
+        redirect_to new_planning_application_assessment_local_policy_path(@planning_application)
+      end
+
+      def create
+        @local_policy_area = @local_policy.local_policy_areas.new(local_policy_params.except(:policy))
+
+        if @local_policy_area.save
+          redirect_to edit_planning_application_assessment_local_policy_path(@planning_application, @planning_application.local_policy)
+        else
+          respond_to do |format|
+            format.html { render :new }
+          end
+        end
+      end
+
+      def update
+        if @local_policy_area.update(local_policy_params)
+          redirect_to edit_planning_application_assessment_local_policy_path(@planning_application, @planning_application.local_policy),
+            notice: I18n.t("local_policies.successfully_updated")
```

Comment:
I think we're using just `t` now rather than `I18n.t` - also other controllers are using relative keys, e.g. `t(".success")`